src/main/scala/com/databricks/spark/sql/perf/Benchmark.scala

```diff
@@ -92,20 +93,20 @@ abstract class Benchmark(
   }
 
   /**
-   * Starts an experiment run with a given set of queries.
-   * @param queriesToRun a list of queries to be executed.
-   * @param includeBreakdown If it is true, breakdown results of a query will be recorded.
+   * Starts an experiment run with a given set of executions to run.
+   * @param executionsToRun a list of executions to run.
+   * @param includeBreakdown If it is true, breakdown results of an execution will be recorded.
    *                         Setting it to true may significantly increase the time used to
-   *                         execute a query.
-   * @param iterations The number of iterations to run of each query.
+   *                         run an execution.
+   * @param iterations The number of iterations to run of each execution.
    * @param variations [[Variation]]s used in this run.  The cross product of all variations will be
-   *                   run for each query * iteration.
+   *                   run for each execution * iteration.
    * @param tags Tags of this run.
    * @return It returns a ExperimentStatus object that can be used to
    *         track the progress of this experiment run.
    */
   def runExperiment(
-      queriesToRun: Seq[Query],
+      executionsToRun: Seq[Benchmarkable],
```

Comment:
BTW, in the future we should avoid changing argument names, even if they aren't totally correct.  This broke a bunch of notebooks that were using named arguments.